### PR TITLE
build: fix CI issues

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

**Review by crisbeto — APPROVED** [review]

**Review by devversion — APPROVED** [review]

File: src/material/tooltip/tooltip.spec.ts [hunk 1]
```diff
@@ -287,7 +301,9 @@ describe('MDC-based MatTooltip', () => {
       tick();
       expect(tooltipDirective._isTooltipVisible()).toBe(true);
 
-      expect(tooltipDirective._overlayRef!.overlayElement.offsetLeft).toBe(triggerRect.right - 100 - 20);
+      expect(tooltipDirective._overlayRef!.overlayElement.offsetLeft).toBe(
+        triggerRect.right - 100 - 20,
```
Comment (josephperrott) [inline src/material/tooltip/tooltip.spec.ts:305]: Is there a reason this isn't just `- 80`?

File: tools/public_api_guard/material/tooltip.md [hunk 2]
```diff
@@ -107,10 +107,19 @@ export abstract class _MatTooltipBase<T extends _TooltipComponentBase> implement
...
+    get positionAtOrigin(): boolean;
+    set positionAtOrigin(value: BooleanInput);
+    show(delay?: number, origin?: {
+        x: number;
+        y: number;
+    }): void;
     get showDelay(): number;
     set showDelay(value: NumberInput);
-    toggle(): void;
+    toggle(origin?: {
```
Comment (josephperrott) [inline tools/public_api_guard/material/tooltip.md:119]: @devversion this is just now showing up, does this mean that our api guard wasn't properly finding the parameters before?

**Review by josephperrott — APPROVED** [review]
LGTM

File: tools/public_api_guard/material/tooltip.md (hunk 2 above)
Comment (devversion) [reply tools/public_api_guard/material/tooltip.md:119]: I think the API file just hasn't been approved/updated in `main` (hence the job being red there)

**Comment by mmalerba** [comment]
I think these failures got in from a PR that didn't trigger our CI jobs for some reason. The PR turned out to be problematic so I reverted it and am closing this PR

**Pull request closed without merging**